File: JTalk/src/Models/Message.java
package Models;

public class Message {

    public Message(int Number, String Text) {
        this.Number = Number;
        this.Text = Text;
    }

    public Message() {
    }

    public int Number;

    public String Text;


    @Override
    public String toString() {
        return Number + " : " + Text;
    }
}

File: TalkSharp/TalkSharp/Models/Message.cs
namespace TalkSharp.Models
{
    /* Use [JsonIgnore] if you need to ignore a field */

    public class Message
    {
        public Message(int Number, string Text)
        {
            this.Number = Number;
            this.Text = Text;
        }

        public int Number { get; set; }

        public string Text { get; set; }
    }
}